instagram/src/App.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import SearchBar from './SearchBar';
+import PostContainer from './PostContainer';
+import postData from './application-data';
+
+class App extends Component {
+  constructor() {
+    super();
+      this.state = {
+        search: 'Search',
+        postData
+      };
+      this.handleSearch = this.handleSearch.bind(this);
+  }
+
+  componentDidMount() {
+    console.log('loaded and ready to go');
+  }
+
+  handleSearch(event) {
```

Comment:
The way you're handling search here won't work I think. You need to split up updating the state that stores the input string and actually performing the submission to clear the input. Here, you're never updating the state, so the search bar will only ever get the input "search".